### DISCUSSION TITLE
TailWind not working with Vite + React

### DISCUSSION THREAD

**Discussion by hosenur** [tailwindlabs/tailwindcss#10219] [Help]
When using TailWind with Vite + React , running the npx tailwindcss init doesnt generate the postcss config, also postcss isn't getting installed . I have followed both guides for React and Vite, setting things up manually gets it to work.

**Comment by wongjn** [comment]
> running the npx tailwindcss init doesnt generate the postcss config

If you follow the [Vite install guide](https://tailwindcss.com/docs/guides/vite), it suggests to run the command:
```sh
npx tailwindcss init -p
```
Notice the `-p` flag which would generated the sample PostCSS configuration for you. Not including this flag would only generate a sample `tailwind.config.js` file.

> also postcss isn't getting installed  

Again, the aforementioned guide for Vite suggests installing dependencies via the command:
```sh
npm install -D tailwindcss postcss autoprefixer
```
This should install the minimum necessary packages, which includes PostCSS.
